pkg/providers/cloudstack/cloudstack.go

```diff
@@ -47,6 +49,9 @@ var defaultClusterConfigMD string
 //go:embed config/machine-health-check-template.yaml
 var mhcTemplate []byte
 
+//go:embed config/secrets.yaml
+var defaultSecretsTemplate string
```

Comment:
it looks like this change will be backwards compatible, right? We're still passing the env var to CAPC in addition to the secret

Reply:
Yes.